scripts/local-airflow.sh

```diff
@@ -16,5 +17,11 @@ ln -s $PWD/$target_dag $AIRFLOW_HOME/dags/
 
 cd $AIRFLOW_HOME
 
+# Disable auth login
+echo "AUTH_ROLE_PUBLIC = 'Admin'" > webserver_config.py
+
 export AIRFLOW__CORE__ALLOWED_DESERIALIZATION_CLASSES_REGEXP='.*'
+export AIRFLOW__CORE__LOAD_EXAMPLES=False
+export AIRFLOW__WEBSERVER__WEB_SERVER_HOST=localhost
+export AIRFLOW__API__AUTH_BACKENDS=airflow.api.auth.backend.default
```

Comment:
What does this do?

Reply:
Interestingly, even with `AUTH_ROLE_PUBLIC = 'Admin'` I wasn't able to view task logs without logging in, the GET requests hit 401 Unauthorized. [airflow.api.auth.backend.default](https://airflow.apache.org/docs/apache-airflow/stable/configurations-ref.html#auth-backends) overrides the local auth backend to allow all requests, which fixes this.